src/styles/index.scss

```diff
@@ -2,2 +1,87 @@
+/* Глобальные настройки */
+* {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: Roboto, sans-serif;
+  background-color: #f9f9f9;
+}
+
+/* Карточки продуктов */
+.product-cards {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  gap: 20px;
+  padding: 20px;
+
+  &__item {
+    width: 200px;
+    box-shadow: 0 4px 6px rgba(0, 0, 0, 0.1);
+
+    &[data-qa='card'] {
+      border-color: #007bff;
```

Comment:
Consider using SCSS variables for colors like '#007bff' to maintain consistency and make future changes easier. Place these variables in a separate file as per the task requirements.